ddcz/views.py

```diff
@@ -639,3 +642,28 @@ def tavern(request):
             "current_list_style": list_style,
         },
     )
+
+
+def sign_up(request):
+    form = SignUpForm()
+
+    if request.method == "POST" and request.POST["submit"]:
+        sgn = SignUpForm(request.POST)
```

Comment:
fwiw, any reason why not to just `form = SignUpForm(request.POST)`? It would spare you the else branche below and I don't see a disadvantage.